wee_slack.py

```diff
@@ -2548,8 +2554,17 @@ def subprocess_thread_message(message_json, eventrouter, channel, team):
 
             text = message.render()
             # channel.buffer_prnt(message.sender, text, message.ts, **kwargs)
-            if parent_message.thread_channel:
+            if parent_message.thread_channel and parent_message.thread_channel.active:
                 parent_message.thread_channel.buffer_prnt(message.sender, text, message.ts, tag_nick=message.sender_plain)
+            elif message.has_mention():
+                if config.auto_open_threads:
```

Comment:
Can you create a function for this block, so you don't have to repeat it?

Reply:
This part:
```
                if config.auto_open_threads:
                     parent_message.open_thread()
                else:
                    team.buffer_prnt(
                            "You were mentionned in thread {}, channel {}".format(
                                parent_message.hash,
                                channel.formatted_name()
                            ), highlight=True)
```

Except for a different message, this is repeated three times.

Reply:
Understood. Something like `parent_message.notify_thread(action=("mention"|"participant"|"response"))` ?

Reply:
Or `parent_message.notify_thread("You were mentionned in bla bla bla")` ?

Reply:
Yeah, the first of those is fine.

Reply:
✓